OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs

```diff
@@ -52,7 +52,8 @@ public GameInfoStatsLogic(Widget widget, World world, OrderManager orderManager)
 				var pp = p;
 				var client = world.LobbyInfo.ClientWithIndex(pp.ClientIndex);
 				var item = playerTemplate.Clone();
-				LobbyUtils.SetupClientWidget(item, client, orderManager, client.Bot == null);
+				if (client != null)
+					LobbyUtils.SetupClientWidget(item, client, orderManager, client.Bot == null);
```

Comment:
`SetupClientWidget` sets the visibility of some widgets. Will these all be visible by default if we skip this call because the client is null? It doesn't look like that would be correct behaviour.